Apps/phreversinglabs/reversinglabs.json

```diff
@@ -4,19 +4,20 @@
     "package_name": "phantom_reversinglabs",
     "description": "This app implements an investigative action on the ReversingLabs reputation service",
     "type": "reputation",
-    "main_module": "reversinglabs_connector.pyc",
-    "app_version": "1.2.30",
-    "utctime_updated": "2020-01-24T01:13:54.000000Z",
+    "main_module": "reversinglabs_connector.py",
+    "app_version": "2.0.0",
+    "utctime_updated": "2019-10-14T23:05:49.000000Z",
     "product_vendor": "ReversingLabs",
     "product_name": "TitaniumCloud",
     "product_version_regex": ".*",
-    "min_phantom_version": "4.2.7532",
+    "min_phantom_version": "4.5.15922",
+    "latest_tested_versions": ["N/A (Note: tested using ReversingLabs TitaniumCloud API as of 1/24/2020)"],
```

Comment:
Can you find out what version it was?

Reply:
I tried, unfortunately it's the cloud api and I can't log in.  I'll look and see if there is a way to look it up.

Reply:
Updated as per @ricardo-phantom format.